tests/static/tests-binder-commonEventBindingsOverride.html

```diff
@@ -0,0 +1,53 @@
+<html>
+    <head>
+        <title>Unit Tests for Overriding Binder Module Common Event Bindings button support...</title>
+
+        <!-- Bring in jQuery and other dependencies -->
+        <script type="text/javascript" src="../../node_modules/infusion/src/lib/jquery/core/js/jquery.js"></script>
+
+        <!-- Bring in fluid dependencies -->
+        <script type="text/javascript" src="../../node_modules/infusion/dist/infusion-all.js"></script>
+
+        <script type="text/javascript" src="../../node_modules/infusion/tests/test-core/utils/js/IoCTestUtils.js"></script>
+
+        <!-- QUnit/jqUnit dependencies -->
+        <script type="text/javascript" src="../../node_modules/infusion/tests/lib/qunit/js/qunit.js"></script>
+        <script type="text/javascript" src="../../node_modules/infusion/tests/test-core/jqUnit/js/jqUnit.js"></script>
+        <script type="text/javascript" src="../../node_modules/infusion/tests/test-core/jqUnit/js/jqUnit-browser.js"></script>
+        <script src="/testem.js"></script>
+
+        <link rel="stylesheet" href="../../node_modules/infusion/tests/lib/qunit/css/qunit.css"/>
+
+        <!-- Our test fixtures -->
+        <script type="text/javascript" src="./js/tests-fixtures.js"></script>
+
+        <!-- Bring in client library being tested -->
+        <script type="text/javascript" src="/coverage/client/coverageSender.js"></script>
+        <script type="text/javascript" src="../../src/js/binder.js"></script>
+        <script type="text/javascript" src="../../src/js/common-event-bindings.js"></script>
+    </head>
+    <body>
+        <h1 id="qunit-header">"Common Event Bindings" Component Tests</h1>
```

Comment:
Please update the page and header title to distinguish this from the other tests.